tutorials/tts/FastPitch_Adapter_Finetuning.ipynb

```diff
@@ -0,0 +1,868 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "207c7f02",
+   "metadata": {},
+   "source": [
+    "## a. Data Preparation\n",
+    "For our tutorial, we use small part of VCTK dataset with a new target speaker (p267). Usually, the audios should have total duration more than 15 mintues."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a7a55ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cd {data_dir} && wget https://vctk-subset.s3.amazonaws.com/vctk_subset.tar.gz && tar zxf vctk_subset.tar.gz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c32a4e5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "manidir = f\"{data_dir}/vctk_subset\"\n",
+    "!ls {manidir}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fab9f979",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_manifest = os.path.abspath(os.path.join(manidir, 'train.json'))\n",
+    "valid_manifest = os.path.abspath(os.path.join(manidir, 'dev.json'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1df877a",
+   "metadata": {},
+   "source": [
+    "## b. Preprocessing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af34a373",
+   "metadata": {},
+   "source": [
+    "### Add absolute file path in manifest\n",
+    "We use absoluate path for audio_filepath to get the audio during training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eb43d862",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo.collections.asr.parts.utils.manifest_utils import read_manifest, write_manifest"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4dcdb0e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_datas = read_manifest(train_manifest)\n",
+    "for m in train_datas: m['audio_filepath'] = os.path.abspath(os.path.join(manidir, m['audio_filepath']))\n",
+    "write_manifest(train_manifest, train_datas)\n",
+    "\n",
+    "valid_datas = read_manifest(valid_manifest)\n",
+    "for m in valid_datas: m['audio_filepath'] = os.path.abspath(os.path.join(manidir, m['audio_filepath']))\n",
+    "write_manifest(valid_manifest, valid_datas)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93c0f45a",
+   "metadata": {},
+   "source": [
+    "### Extract Supplementary Data\n",
+    "\n",
+    "As mentioned in the [FastPitch and MixerTTS training tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_MixerTTS_Training.ipynb) - To accelerate and stabilize our training, we also need to extract pitch for every audio, estimate pitch statistics (mean, std, min, and max). To do this, all we need to do is iterate over our data one time, via `extract_sup_data.py` script.\n",
+    "\n",
+    "Note: This is an optional step, if skipped, it will be automatically executed within the first epoch of training FastPitch."
```

Comment:
Some past tutorials said this, but it is not entirely correct. If the script is not run then the training script will compute the pitch during first epoch but it will either not normalize the pitch, or normalize using default pitch values (eg. hardcoded from LJSpeech).

For most of our example Fastpitch configs we now set `pitch_mean: ???` so that user cannot accidentally run the training without precomputing and providing the statistics.